app/gateway/changelog.md

```diff
@@ -196,6 +243,9 @@ Consumer entity, and the username would remain locked. For example, if the
 admin was created in workspace `dev` and deleted from workspace `QA`, this
 issue would occur.
 
+* Phone home metrics are now sent over TLS, meaning that any analytics data
+on Kong Gateway usage now travels through an encrypted connection.
```

Comment:
Note that I also added this change to the 2.7.0.0 changelog, since it got missed.